packages/expo-updates/android/src/main/java/expo/modules/updates/manifest/raw/RawManifest.kt

```diff
@@ -149,7 +149,7 @@ abstract class RawManifest(protected val json: JSONObject) {
   fun getAndroidUserInterfaceStyle(): String? {
     val expoClientConfig = getExpoClientConfigRootObject() ?: return null
     val android = expoClientConfig.optJSONObject("android") ?: return null
```

Comment:
could we fall back to `expoClientConfig.optString("userInterfaceStyle")` here as well?

Reply:
Right, Thanks! Pushed a new commit, this uses the same approach as in other similar functions (eg with `backgroundColor`)